@navikt/core/css/darkside/expansioncard.darkside.css

```diff
@@ -1,235 +1,187 @@
+  &:focus-visible {
+    outline: 2px solid var(--ax-border-focus);
+    outline-offset: 2px;
   }
-}
-
-.navds-expansioncard--open
-  > :where(.navds-expansioncard__header)
-  > :where(.navds-expansioncard__header-button):hover
-  :where(.navds-expansioncard__header-chevron) {
-  transform: translateY(-1px) rotate(180deg);
-}
 
-.navds-expansioncard__header-button::after {
-  inset: 0;
-  z-index: 1;
-  position: absolute;
-  content: "";
-  height: 100%;
-  width: 100%;
-  cursor: pointer;
-}
-
-/*************************
- * Content *
- *************************/
-.navds-expansioncard__content {
-  --__ac-expansioncard-padding-block: var(--a-spacing-5);
-
-  border-end-end-radius: var(--__ac-expansioncard-border-radius);
-  border-end-start-radius: var(--__ac-expansioncard-border-radius);
-  padding: var(--__ac-expansioncard-padding-block) var(--__ac-expansioncard-padding-inline)
-    var(--__ac-expansioncard-padding-inline);
-  border: var(--__ac-expansioncard-border-width) solid var(--__ac-expansioncard-border-color);
-  border-top: none;
-}
+  @supports not selector(:focus-visible) {
+    &:focus {
+      outline: 2px solid var(--ax-border-focus);
+      outline-offset: 2px;
+    }
+  }
 
-.navds-expansioncard--small .navds-expansioncard__content {
-  --__ac-expansioncard-padding-block: var(--a-spacing-4);
+  /* Makes the whole header clickable */
+  &::after {
+    inset: 0;
+    z-index: 1;
+    position: absolute;
+    content: "";
+    height: 100%;
+    width: 100%;
+    cursor: pointer;
+  }
 }
 
-.navds-expansioncard__content--closed {
-  display: none;
+.navds-expansioncard__header:hover > .navds-expansioncard__header-button {
+  background-color: var(--ax-bg-neutral-moderate-hoverA);
 }
 
-:where(.navds-expansioncard__header):hover + .navds-expansioncard__content {
-  box-shadow:
-    1px 0 0 0 var(--__ac-expansioncard-border-color),
-    -1px 0 0 0 var(--__ac-expansioncard-border-color),
-    0 1px 0 0 var(--__ac-expansioncard-border-color);
+.navds-expansioncard__header-chevron {
+  transition: transform 250ms cubic-bezier(0.2, 0, 0, 1);
 }
 
-.navds-expansioncard__content-inner {
-  animation: fadeExpansionCard 250ms ease;
+.navds-expansioncard__header[data-open="true"] .navds-expansioncard__header-chevron {
+  transform: rotateX(-180deg);
 }
 
-.navds-expansioncard--no-animation :where(.navds-expansioncard__content-inner) {
-  animation: none;
+.navds-expansioncard__header[data-open="false"] .navds-expansioncard__header-chevron {
+  transition-duration: 0ms;
 }
 
-@keyframes fadeExpansionCard {
-  0% {
-    opacity: 0.25;
-    transform: translateY(-8px);
-  }
+/* -------------------------- ExpansionCard Content ------------------------- */
+.navds-expansioncard__content {
+  --__axc-expansioncard-padding-block: var(--ax-spacing-5);
 
-  40% {
-    opacity: 0.7;
+  border-end-end-radius: var(--__axc-expansioncard-border-radius);
+  border-end-start-radius: var(--__axc-expansioncard-border-radius);
```

Comment:
Is this border stuff needed? If not, the variable can probably also be removed.

Reply:
Its needed in the off-chance one adds a background-color to content:
<img width="145" alt="Screenshot 2024-11-13 at 11 53 35" src="https://github.com/user-attachments/assets/771ffbcb-1bf1-4531-b95c-2fe74b1febc8">
<img width="67" alt="Screenshot 2024-11-13 at 11 55 57" src="https://github.com/user-attachments/assets/c7764f7a-886f-43f1-9db8-ff38c23b4464">

But made me find a px-bug so changed it to `calc(var(--ax-border-radius-xlarge) - 1px);` to avoid the small gap in the corners.